### PR TITLE
[Feature] Custom container plugin 

### PR DESCRIPTION
**Pull request by antoine4livre**
Linked issue : #62 

### Usage:
```js
const { config } = require('@studiometa/tailwind-config');

module.exports = {
  presets: [config],
  theme: {
    extend: {
      customContainer: {
        padding: {
          DEFAULT: '1rem',
          m: '2rem',
        },
        maxWidth: {
          DEFAULT: '80rem',
          xxl: '94rem',
        },
      },
    }
  },
};
```

### Available properties
- center (not responsive): used to center the container
- padding (responsive)
- maxWidth (responsive)

### Default config:
- center: true
- padding: 1rem
- maxWidth: 80rem

### Use the default container corePlugin
```js
const { config } = require('@studiometa/tailwind-config');

module.exports = {
  presets: [config],
  theme: {
    customContainer: false,
    container: {
      center: true,
    },
  },
  corePlugins: {
    container: true,
  },
};
```

### CODE REVIEW

**Comment by titouanmathis** [comment]
Just to be sure: by default the container from Tailwind is disabled and our custom one is enabled with the mentioned default values. Is it correct?

File: packages/tailwind-config/plugins/custom-container.js [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+            propertyValueForCurrentBreakpoint;
+        }
+      );
+    });
+
+  const container = { ...containerDefault, ...containerMediaQueries };
+
+  addUtilities({
+    '.container': container,
+    '.container-reset': Object.fromEntries(Object.keys(container).map((key) => [key, 'unset'])),
```
Comment (titouanmathis) [inline packages/tailwind-config/plugins/custom-container.js:72]: **question:** What would be the usage for the reset?

Comment (titouanmathis) [reply packages/tailwind-config/plugins/custom-container.js:72]: I have the answer just below, nevermind.

**Review by titouanmathis — APPROVED** [review]
Ok for me, go for the doc 👍

File: packages/docs/configuration/readme.md [hunk 2]
```diff
@@ -155,11 +155,31 @@ module.exports = {
 };
 ```
 
+### Custom container
```
Comment (titouanmathis) [inline packages/docs/configuration/readme.md:158]: **chore :** Move this section to the top to replace the current `### Container` section

**Review by titouanmathis — CHANGES_REQUESTED** [review]

File: packages/docs/configuration/readme.md (hunk 2 above)
Comment (antoine4livre) [reply packages/docs/configuration/readme.md]: Done ✅

**Pull request merged**